src/plugins/intel_gpu/docs/simplified_workflow.md

```diff
@@ -3,128 +3,110 @@
 The simplified workflow in the GPU plugin is shown in the diagram below (click it for higher resolution):
 
 ```mermaid
-classDiagram 
-class `intel_gpu::Plugin` {Inference Engine plugin 
-implementation for GPU}
-class `intel_gpu::CompiledModel` {Device specific network 
-representation that can be executed}
-class `intel_gpu::InferRequestAsync` {
-Inference request for specific executable network. 
-Wrapper for input and output memory}
-class `intel_gpu::TransformationPipeline` {Set of ngraph-based transformations 
-configured by GPU plugin}
-`Core::compile_model()` --> `intel_gpu::CompiledModel`
-`CompiledModel::create_infer_request()` -->`intel_gpu::InferRequestAsync`
-`InferRequest::start_async()` --> `intel_gpu::network`
-`intel_gpu::Plugin` --|> `InferenceEngine::InferencePluginInternal`
+classDiagram
```

Comment:
Maybe we can split this diagram on several parts? Because here we have a lot of connections and it looks unreadable.
<img width="1558" alt="Screenshot 2023-08-28 at 07 22 53" src="https://github.com/openvinotoolkit/openvino/assets/4737252/1ffd9ec5-593f-4903-a075-41e8b7f8b7bb">

Reply:
Yes, we need to do something with this diagram, as it's unreadable in this format. Anyway, it's not a scope of this PR.